internal/httpserve/handlers/login.go

```diff
@@ -74,7 +74,8 @@ func (h *Handler) LoginHandler(ctx echo.Context) error {
 	props := ph.NewProperties().
 		Set("user_id", user.ID).
 		Set("email", user.Email).
-		Set("organization_id", claims.OrgID)
+		Set("organization_id", claims.OrgID).
+		Set("auth_provider", "CREDENTIALS")
```

Comment:
Can you use `user.AuthProvider` instead of hardcoding? 
```suggestion
		Set("auth_provider", user.AuthProvider)
```